examples/json_validation/README.md

```diff
@@ -54,6 +57,7 @@ This example details deploying to a kind cluster running the Istio httpbin sampl
 kind create cluster
 
 # Install Istio and the httpbin sample app
+
 istioctl install --set profile=demo -y
```

Comment:
@liamawhite could you please test from here on? I am in trouble to run this in my M1 mac now.